ZenPacks/zenoss/LinuxMonitor/LogicalVolume.py

```diff
@@ -62,3 +62,11 @@ def openstack_core_components(self):
             return get_cinder_components(self)
         else:
             return []
+
+    def filesystem(self):
+        results = ICatalogTool(self.device()).search('Products.ZenModel.FileSystem.FileSystem')
```

Comment:
Similar to my note on FileSystem.getLogicalVolume() above, it's not a good idea to use the global catalog when searching for things local to a device. Unfortunately in this case, we couldn't use zenpacklib to create the FileSystem class. So we can't easily index any of its properties. The best thing we can do here is just iterate through each of the device's filesystems.

```python
for fs in self.device().os.filesystems():
    if fs.title == self.mountpoint:
        return fs
```

No need to explicitly return None.